diff -purN 2.0.2.0/src/vos/posix/vos_thread.c 2.0.2.0/src/vos/posix/vos_thread.c
--- 2.0.2.0/src/vos/posix/vos_thread.c	2020-03-03 11:28:52.888000000 +0100
+++ trdp-2.0.0.1/tcnopen-trdp/2.0.2.0/src/vos/posix/vos_thread.c	2020-03-03 13:05:31.928000000 +0100
@@ -58,7 +58,7 @@
 #ifdef __APPLE__
 #include <uuid/uuid.h>
 #else
-#include <uuid.h>
+#include <uuid/uuid.h>
 #endif
 #else
 #include "vos_sock.h"
